src/views/parents/l10n.json

```diff
@@ -0,0 +1,26 @@
+    "parents.ageRangeDescription": "While Scratch is primarily designed for 8 to 16 year olds, it is also used by people of all ages, including younger children with their parents.",
+    "parents.resources": "What resources are available for learning Scratch?",
+    "parents.resourcesDescription": "If you’re just getting started, there’s a <a href=\"/projects/editor/?tip_bar=getStarted\">step-by-step guide</a> available inside Scratch, or you can download the <a href=\"https://cdn.scratch.mit.edu/scratchr2/static/__2da8c1b247407cc9fe0b4c4504bed936__//pdfs/help/Getting-Started-Guide-Scratch2.pdf\">Getting Started guide (PDF)</a>. The <a href=\"/cards\">Scratch Cards</a> provide a fun way to learn more. For an overview of Scratch resources, see <a href=\"/help\">Scratch Help</a>.",
+    "parents.onlineCommunity": "What is the Scratch online community?",
+    "parents.onlineCommunityDescription": "When participating in the Scratch online community, members can explore and experiment in an open learning community with other Scratch members from all backgrounds, ages, and interests. Members can share their work, get feedback, and learn from each other.",
+    "parents.guidelines": "What are the guidelines for the Scratch online community?",
+    "parents.guidelinesDescription": "The MIT Scratch Team works with the community to maintain a friendly and respectful environment for people of all ages, races, ethnicities, religions, sexual orientations, and gender identities. You can help your child learn how to participate by reviewing the <a href=\"/guidelines\">Community Guidelines</a> together. Members are asked to comment constructively and to help keep the website friendly by reporting any content that does not follow the community guidelines. The Scratch Team works each day to manage activity on the site and respond to reports, with the help of tools such as the <a href=\"https://www.inversoft.com/products/profanity-filter\">CleanSpeak</a> profanity filter.",
+    "parents.privacyPolicy": "What is your privacy policy?",
+    "parents.privacyPolicyDescription": "To protect children's online privacy, we limit what we collect during the signup process, and what we make public on the website. We don't sell or rent account information to anyone. You can find out more about our privacy policy on our <a href=\"/privacypolicy\">Privacy Policy</a> page.",
+    "parents.noOnline": "Is there a way to use Scratch without participating online?",
+    "parents.noOnlineDescription": "Yes, the Scratch offline editor lets you create projects without joining or accessing the online community. Visit the <a href=\"/scratch2download/\">Scratch 2.0 offline editor</a> download page for instructions on how to install it on your computer. (If your computer does not support the latest version, try the <a href=\"/scratch_1.4/\">Scratch 1.4 offline editor</a>.)",
+    "parents.parentsSaying": "Learn More About Scratch",
```

Comment:
On the current parents page I think this says "What are parents saying about Scratch?"